guides/advanced_search_guide.ipynb

```diff
@@ -1,762 +1,1474 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    docs[0].keys()
</pre><p>Don't need this either :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RelevanceAI/RelevanceAI/pull/673/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>